ptftplib/tftpserver.py

```diff
@@ -70,9 +70,11 @@ def get_ip_config_for_iface(iface):
 
 def get_max_udp_datagram_size():
     """Retrieve the maximum UDP datagram size allowed by the system."""
-    val = subprocess.check_output(['sysctl', '-n', 'net.inet.udp.maxdgram'])
-    return int(val)
-
+    try:
+        val = subprocess.check_output(['sysctl', '-n', 'net.inet.udp.maxdgram'])
+        return int(val)
+    except subprocess.CalledProcessError:
+        return 508
```

Comment:
Could you define a constant for this "magic" value, and document that constant to explain the choice of this particular value for a fallback UDP datagram size?

Reply:
Thanks for getting back to me. I'll fix this up. :)

Reply:
@fifteenhex did you get a chance to update this? Would love to merge this PR!